templates/store/snap-details.html

```diff
@@ -158,6 +158,18 @@
       <div class="col-4">
         {% include "store/snap-details/_details.html" %}
 
+        {% if request.path in ["/vlc", "/standard-notes", "/plexmediaserver"] %}
+        <h4>Companion apps</h4>
+        <ul class="p-list">
+          <li class="p-list__item u-no-padding--left"><a href="{% if request.path == '/vlc' %}https://apps.apple.com/us/app/vlc-for-mobile/id650377962{% elif  request.path == '/standard-notes' %}https://apps.apple.com/gb/app/standard-notes/id1285392450{% elif request.path == '/plexmediaserver' %}https://apps.apple.com/gb/app/plex-movies-tv-music-more/id383457673{% endif %}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : '3rd Party Store Link', 'eventAction' : 'App Store Link ', 'eventLabel' : 'App Store Link', 'eventValue' : undefined });"><img
```

Comment:
Could "eventValue" maybe be the snap name? Would that be useful? Not sure
Also perhaps split these lines as they get pretty long